user/test.py

```diff
@@ -0,0 +1,123 @@
+from django.test import TestCase
+from django.contrib.auth.forms import UserCreationForm, AuthenticationForm
```

Comment:
```suggestion
from django.contrib.auth.forms import AuthenticationForm
```
使っていないモジュールは消しちゃってください。